e2e/cypress/support/ui_commands.js

```diff
@@ -55,6 +55,18 @@ Cypress.Commands.add('uiChangeMessageDisplaySetting', (setting = 'STANDARD') =>
     cy.get('#accountSettingsHeader > .close').click();
 });
 
+Cypress.Commands.add('verifyAccountNameSettings', (firstname, lastname) => {
```

Comment:
We're organizing better the commands. With that, please move this to `e2e/cypress/support/ui/account_settings_modal.js`. Same with its corresponding typing from `.d.ts`.